journal/modernized/26-chapter-25.adoc

```diff
@@ -1,15 +1,15 @@
-== Section XXV.
+== Chapter XXV
 
-In 1704, he, according to his usual custom,
-visits Friends in their National and Provincial Meetings;
-afterwards in his sickness is resigned to the will of God,
-and gives a summary of his life, faith and practice, etc.
+[.chapter-synopsis]
+* In 1704, he visits Friends in their National and Provincial Meetings;
+* In his sickness he is resigned to the will of God and gives a summary of his life, faith, and practice, etc.
```

Comment:
Yup, a little liberty probably was necessary here.  Let's make two small tweaks, first remove the semi-colon, and then let's break out one more line, here's what I propose changing it to:

```ASCIIDOC
* In 1704, he visits Friends in their National and Provincial Meetings
* In his sickness is resigned to the will of God
* Gives a summary of his life, faith, and practice, etc.
```